lib/improver/spotdata/site_data.py

```diff
@@ -147,9 +147,9 @@ def parse_input(self, site_data):
 
         """
         latitude_entries = [i_site for (i_site, site) in enumerate(site_data)
-                            if 'latitude' in site.keys()]
+                            if 'latitude' in list(site.keys())]
```

Comment:
`if 'latitude' in site` same thing below

Reply:
Done.